tests/ui/rust-2024/unsafe-extern-blocks/safe-unsafe-on-unadorned-extern-block.stderr

```diff
@@ -0,0 +1,14 @@
+error: safety is not expected in unadorned extern blocks
+  --> $DIR/safe-unsafe-on-unadorned-extern-block.rs:2:5
+   |
+LL |     safe static TEST1: i32;
+   |     ^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
If possible, it'd be good to have a machine-applicable suggestion here to add `unsafe` to the surrounding `extern` block.